src/pagination/pagination.ts

```diff
@@ -11,39 +11,43 @@ import {NgbPaginationConfig} from './pagination-config';
   host: {'role': 'navigation'},
   template: `
     <ul [class]="'pagination' + (size ? ' pagination-' + size : '')">
-      <li *ngIf="boundaryLinks" class="page-item"
-        [class.disabled]="!hasPrevious() || disabled">
-        <a aria-label="First" i18n-aria-label="@@ngb.pagination.first-aria" class="page-link" href
-          (click)="!!selectPage(1)" [attr.tabindex]="(hasPrevious() ? null : '-1')">
+      <li *ngIf="boundaryLinks" title="{{!hasPrevious() || disabled ? 'First Page (disabled)' : 'First Page'}}" class="page-item"
```

Comment:
- if you're adding a title attribute, you should also add `i18n` support for it
- why on `li`, and not on the `a` tag as other attributes?